apps/server/src/modules/learnroom/mapper/room-board-response.mapper.spec.ts

```diff
@@ -65,6 +65,33 @@ describe('room board response mapper', () => {
 			expect(result.elements[0] instanceof BoardElementResponse).toEqual(true);
 		});
 
+		it('should map tasks with status and its task card on board to response', () => {
+			const course = courseFactory.buildWithId();
+			const taskCard = taskCardFactory.buildWithId({ course });
+			const linkedTask = taskCard.task;
+			linkedTask.course = course;
+			linkedTask.taskCard = taskCard.id;
```

Comment:
```suggestion
			const linkedTask = taskFactory.buildWithId({ course });
			linkedTask.taskCard = 'foo';
```
the task is dumb, does not know it must be created from task-card
the only info we have for it, is that it has a taskCardId string